bin/mail/2016-09-27-hoc-volunteer-launch/generate-mailing-list.rb

```diff
@@ -2,5 +2,12 @@
 require_relative '../mailing-common/mailing-list-utils'
 
 contacts = query_subscribed_contacts q: 'kind_s:VolunteerEngineerSubmission2015'
+
+# Add name_s field, copied from name, because name_s is referenced in the template and name is stripped out here:
+# https://github.com/code-dot-org/code-dot-org/blob/cefeb5a0f90b06a65281187ef3ff40b5eb93d83e/bin/mail/send-to-mailing-list#L77
+contacts.values.each do |contact|
+  contact[:name_s] = contact[:name]
+end
```

Comment:
LGTM. Though I'm concerned that we are approaching this the wrong way, especially if there is legitimate reason to be stripping the `contact[:name]` via the link. This essentially bypasses the value of that stripping.

Please confirm there isn't a privacy reason we need to strip the name (email) from the hash.